pkg/kudoctl/cmd/init.go

```diff
@@ -98,6 +101,7 @@ func newInitCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f.BoolVar(&i.crdOnly, "crd-only", false, "Add only KUDO CRDs to your cluster")
 	f.BoolVarP(&i.wait, "wait", "w", false, "Block until KUDO manager is running and ready to receive requests")
 	f.Int64Var(&i.timeout, "wait-timeout", 300, "Wait timeout to be used")
+	f.StringVarP(&i.serviceaccount, "serviceaccount", "", "", "Override for the default serviceAccount kudo-manager")
```

Comment:
Please change this to service-account -- convention is to add a dash between words in other flags, so we'll prefer sticking with that convention. :) Also, for the description, something along the lines of "Override the default service account for the KUDO controller" is a little more clear.